docs/web-apps/automated-testing/_partials/_advanced-cypress.md

```diff
@@ -32,4 +32,8 @@ When running Cypress test along with Cucumber, you can use tags to select which
 
 ```
 saucectl run --env CYPRESS_TAGS="(@smoke or @ui) and (not @slow)"
-```
\ No newline at end of file
+```
+
+## Cypress Component Test
```

Comment:
```suggestion
## Component Testing
```
I think qualifying it as "Cypress" Component Testing is maybe redundant since this doc is exclusively about cypress already.